pattern-library/js/edx-icons.js

```diff
@@ -1,9 +1,13 @@
-define(['afontgarde'],
-    function() {
-        'use strict';
+define(['afontgarde'], function() {
+    'use strict';
 
-        window.AFontGarde('edx-icons', {
-            // Check a few random icons to see if our font loads
-            glyphs: '\uE621\uE622\uE623'
-        });
+    // Old icon fonts, currently used in the platform.
+    window.AFontGarde('FontAwesome', {
```

Comment:
I think of platform as being files in edx-platform. This file will be used by any edX IDAs that want consistent styling, e.g. Insights.